src/main/java/first/controllers/StudentControllerInter.java

```diff
@@ -0,0 +1,16 @@
+package first.controllers;
+
+public interface StudentControllerInter {
+    String str = " <head>\n" +
```

Comment:
Also, constants naming. It should has name STR. But it still bad name. Use something like BASE_ACTIONS_FORM